syntax = "proto3";

package wallet;

service Wallet {
    rpc TopUp(TopupReq) returns (StatusRes) {}
}

message TopupReq {
    string phone_number = 1;
    string fromBank = 2;
    int32 amount = 3;
}

message StatusRes {
    string status = 1;
}